### PR TITLE
Add refresh button for skins, closes #2979

### LINKED ISSUE #2979: Add Refresh button on Skins page
I noticed in the new version there was a Refresh button for Assets such as game.png, entities, etc.

I think it'll be helpful to have the same button for skins so people can easily import and test skins without having to relog the client.

### PR DESCRIPTION
**Pull request by Blacksky780i**
image

### CODE REVIEW

**Comment by def-** [comment]
Oh, I like that refresh unicode symbol. You should probably make a second PR to change all occurences of "Refresh" to that.

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -699,6 +700,16 @@ void CMenus::RenderSettingsTee(CUIRect MainView)
 			dbg_msg("menus", "couldn't open link");
 		}
 	}
+
+	TextRender()->SetCurFont(TextRender()->GetFont(TEXT_FONT_ICON_FONT));
+	TextRender()->SetRenderFlags(ETextRenderFlags::TEXT_RENDER_FLAG_ONLY_ADVANCE_WIDTH | ETextRenderFlags::TEXT_RENDER_FLAG_NO_X_BEARING | ETextRenderFlags::TEXT_RENDER_FLAG_NO_Y_BEARING | ETextRenderFlags::TEXT_RENDER_FLAG_NO_OVERSIZE);
+	if(DoButton_Menu(&RefreshButton, "\xEE\x97\x95", 0, &RefreshButton))
```
Comment (def-) [inline src/game/client/components/menus_settings.cpp:706]: I guess we should at least add a comment showing the unicode symbol, makes it easier for grepping too.

File: src/game/client/components/menus_settings.cpp [hunk 2]
```diff
@@ -699,6 +700,16 @@ void CMenus::RenderSettingsTee(CUIRect MainView)
 			dbg_msg("menus", "couldn't open link");
 		}
 	}
+
+	TextRender()->SetCurFont(TextRender()->GetFont(TEXT_FONT_ICON_FONT));
+	TextRender()->SetRenderFlags(ETextRenderFlags::TEXT_RENDER_FLAG_ONLY_ADVANCE_WIDTH | ETextRenderFlags::TEXT_RENDER_FLAG_NO_X_BEARING | ETextRenderFlags::TEXT_RENDER_FLAG_NO_Y_BEARING | ETextRenderFlags::TEXT_RENDER_FLAG_NO_OVERSIZE);
```
Comment (def-) [inline src/game/client/components/menus_settings.cpp:705]: @Jupeyy Does this collide with your https://github.com/ddnet/ddnet/pull/3022 ?

Comment (Jupeyy) [reply src/game/client/components/menus_settings.cpp:705]: i'll just fix it later, np

File: src/game/client/components/menus_settings.cpp (hunk 1 above)
Comment (Jupeyy) [reply src/game/client/components/menus_settings.cpp:706]: its material icon

**Comment by def-** [comment]
Btw, you shouldn't use master branch for pull requests but create a separate branch for each PR.

Tested and works fine for me. Thanks!

bors r+

**Comment by def-** [comment]
bors retry

**Pull request merged**